.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: emacs-lisp
+sudo: false
+cache: apt
+env:
+  - EVM_EMACS=emacs-25.1-travis
+  - EVM_EMACS=emacs-25.2-travis
```

Comment:
Can we add support for latest `HEAD` Emacs, too?  Does EVM have development versions?